src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/LogoutWithoutSessionInvalidation/config.yml

```diff
@@ -21,4 +21,3 @@ security:
                 secret: secret
             logout:
                 invalidate_session: false
-            stateless: true
```

Comment:
I think this was a mistake.
Causing [this test](https://github.com/symfony/symfony/blob/b7eb42e504de4f7bad9809ce67a2c672c5f5301c/src/Symfony/Bundle/SecurityBundle/Tests/Functional/LogoutTest.php#L23) to fail with my change.

Reply:
maybe @xabbuh remembers?

Reply:
I don't remember, but I guess we just copied an existing config and didn't notice.

Reply:
Hi @chalasr, you resolved the conversation without comment.
Any concern on that PR?